source/uri-options/uri-options.md

```diff
@@ -92,7 +92,7 @@ to URI options apply here.
 | proxyPort                            | non-negative integer                                                                                                                                                                      | defined in the [SOCKS5 support spec](../socks5-support/socks5.rst#mongoclient-configuration)                                                                                                                                 | no                                                                                                                                                                                                          | The port of the SOCKS5 proxy server specified in `proxyHost`.                                                                                                                                                                                                                                    |
 | proxyUsername                        | any string                                                                                                                                                                                | defined in the [SOCKS5 support spec](../socks5-support/socks5.rst#mongoclient-configuration)                                                                                                                                 | no                                                                                                                                                                                                          | The username for username/password authentication to the SOCKS5 proxy server specified in `proxyHost`.                                                                                                                                                                                           |
 | proxyPassword                        | any string                                                                                                                                                                                | defined in the [SOCKS5 support spec](../socks5-support/socks5.rst#mongoclient-configuration)                                                                                                                                 | no                                                                                                                                                                                                          | The password for username/password authentication to the SOCKS5 proxy server specified in `proxyHost`.                                                                                                                                                                                           |
```

Comment:
Are we tracking these links that we'll have to come back and fix later?

Reply:
It will be done automatically when the socks5 spec is migrated.